src/libraries/Native/AnyOS/System.Globalization.Native/pal_icushim_internal.h

```diff
@@ -131,13 +148,14 @@
     FOR_ALL_UNCONDITIONAL_ICU_FUNCTIONS \
     PER_FUNCTION_BLOCK(ucol_setMaxVariable, libicui18n)
 #else
+
 #define FOR_ALL_ICU_FUNCTIONS \
     FOR_ALL_UNCONDITIONAL_ICU_FUNCTIONS \
     PER_FUNCTION_BLOCK(ucol_setVariableTop, libicui18n)
 #endif
 
 // Declare pointers to all the used ICU functions
-#define PER_FUNCTION_BLOCK(fn, lib) extern __typeof(fn)* fn##_ptr;
+#define PER_FUNCTION_BLOCK(fn, lib) EXTERN_C __typeof(fn)* fn##_ptr;
```

Comment:
Why does this needs to be `EXTERN_C` ?

Reply:
This needs to be extern c so that the linker can find the exported symbols. Otherwise I can make them static, what do you suggest?

Reply:
The linker on Unix can find it. How does the Windows linker differ?

This does not seems to build anyway, so let's see this will end being once the build breaks are fixed.